beeline-spring-boot-sleuth-starter/src/test/java/io/honeycomb/beeline/spring/beans/BraveBeelineReporterTest.java

```diff
@@ -71,7 +71,7 @@ public void GIVEN_span_EXEPECT_event() {
 
         final ResolvedEvent captured = (ResolvedEvent) captor.getValue();
         Assert.assertEquals("testKey", captured.getWriteKey());
-        Assert.assertEquals("testSet", captured.getDataset());
+        Assert.assertEquals("unknown_service", captured.getDataset());
```

Comment:
I think it might be worth adding a service name and asserting it's set correctly to dataset, since the goal of this assertion would be that we get a dataset. It may be separately worth seeing that the default gets set to `unknown_service` but that seems less important here.

```suggestion
        Assert.assertEquals("testName", captured.getDataset());
```

Up in line 63, we can add `.serviceName("testName")`:
```java
beeline = spy(beelineBuilder.transport(mockTransport).writeKey("testKey").dataSet("testSet").serviceName("testName").build());
```